src/UnitTests/Analysis/OutParameterTransformerTests.cs

```diff
@@ -69,6 +69,11 @@ private void PerformTest(FileUnitTester fut)
 
 				proc.Write(false, fut.TextWriter);
 				fut.TextWriter.WriteLine("====================");
+                //$REVIEW: OutpMutual test failed. OutParameterTransformer
+                // removes phi assignments from uses. Is it correct, John?*/
+                if (proc.Name == "fn0C00_0004")
```

Comment:
The phi assignment `x_2 = φ(x_0,x_1)` should indeed be a use of both `x_0` and `x_1`.